### DISCUSSION TITLE
启动运行到某一行立刻退出

### DISCUSSION THREAD

**Discussion by Nirlvy** [LmeSzinc/AzurLaneAutoScript#3245] [Q&A]
![IMG_20231030_183843.jpg](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/52414613/c20012b6-2141-4506-bb67-ce92b436bd0a)
到这立刻退出，后台只有adb在运行没有python和alas进程，怀疑和
![Screenshot_2023-10-30-18-40-41-793_com.tencent.mobileqq.png](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/52414613/4f0257ad-363d-4804-866b-fc105014dcd9)
有关但是这个更新无法卸载
有什么办法能追查问题吗

**Comment by LmeSzinc** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/FAQ_en_cn#alas-%E5%BC%95%E5%AF%BC%E5%AE%8C%E6%88%90%E5%90%8E%E9%97%AA%E9%80%80--alas-crashes-after-boot-completion-2481
